<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1071</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>mysql指令</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1041</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>120</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>送出</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>1041</width>
        <height>341</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>商品查詢</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>商品名稱</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>查詢</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>1051</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>購物清單</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>買家ID</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品ID</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_5">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>數量</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>106</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>新增</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>買家ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>106</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>查詢</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>1041</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>480</y>
        <width>106</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>新增至訂單</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>106</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>刪除</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>480</y>
        <width>241</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>除了所選以外的購物清單皆</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>訂單查詢與評分</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>買家ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>106</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>查詢</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>分數</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>70</y>
        <width>106</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>評價</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>1051</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>訂單ID</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>過濾</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>訂單明細</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>買家ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>106</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>查詢</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>651</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_6">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>90</y>
        <width>391</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>最大金額：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>最小金額：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_max">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_min">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>60</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_low">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_up">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>133</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>金額介於</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>~</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>平均金額：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_avg">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>scoreOrder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>906</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>searchOrder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>881</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addShoppingList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>828</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>874</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>searchShoppingList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>916</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>searchByProductName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>856</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getsql()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>886</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addOrder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>901</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteShoppingList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>887</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>searchDetails()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>872</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getsql()</slot>
  <slot>searchByProductName()</slot>
  <slot>addShoppingList()</slot>
  <slot>searchShoppingList()</slot>
  <slot>addOrder()</slot>
  <slot>deleteShoppingList()</slot>
  <slot>searchOrder()</slot>
  <slot>scoreOrder()</slot>
  <slot>searchDetails()</slot>
 </slots>
</ui>
